### PR TITLE
[12.0][FIX] excel_import_export, problem on import with datetime field

### PR DESCRIPTION
**Pull request by kittiu**
Fix for case import if the import file has datetime field

### CODE REVIEW

**Comment by kittiu** [comment]
Note: Travis seem to fail because other modules (OCB).

**Review by JordiBForgeFlow — APPROVED** [review]
:+1: Works as expected.

**Comment by JordiBForgeFlow** [comment]
@kittiu can you rebase? We have tried to greenify the repo here https://github.com/OCA/server-tools/pull/1789

**Comment by kittiu** [comment]
> @kittiu can you rebase? We have tried to greenify the repo here #1789

Done, but seem still one issue with OCB.

**Comment by MiquelRForgeFlow** [comment]
@kittiu @JordiBForgeFlow now it's green :green_apple:

File: excel_import_export/models/common.py [hunk 1]
```diff
@@ -268,13 +268,12 @@ def _get_cell_value(cell, field_type=False):
     datemode = 0  # From book.datemode, but we fix it for simplicity
     if field_type in ['date', 'datetime']:
         ctype = xlrd.sheet.ctype_text.get(cell.ctype, 'unknown type')
-        if ctype == 'number':
+        if ctype in ('xldate', 'number'):
+            is_datetime = cell.value % 1 != 0.0
```
Comment (MiquelRForgeFlow) [inline excel_import_export/models/common.py:272]: I have a question. This kind of check is safe? Cannot fail? Isn't it better to check `field_type == 'datetime'`? Just asking.

**Review by MiquelRForgeFlow — APPROVED** [review]
LGTM

File: excel_import_export/models/common.py (hunk 1 above)
Comment (kittiu) [reply excel_import_export/models/common.py:272]: I am not too certain. But as I saw in Libre office, for the field what is Date, i.e., 2010-10-10, when chagned cell type to text, number or general, it is changed to 40461. So, I just keep that on the safe side (some how).
WDYT?

Comment (kittiu) [reply excel_import_export/models/common.py:272]: Also, comparing to 'number' used to work on my older versions (back in v8).

Comment (MiquelRForgeFlow) [reply excel_import_export/models/common.py:272]: ok

**Comment by kittiu** [comment]
@JordiBForgeFlow can you merge this? I will also fix in v13 too (now it is PR).

**Pull request merged**